### PR TITLE
Feat/11 specify sheets to fetch

### LINKED ISSUE #11: Specify sheets to fetch
Allow set custom set of sheets to fetch

### PR DESCRIPTION
**Pull request by horaklukas**
Adds option to specify which sheets from spreadsheet are supossed to use for generating translations. This change precedes to #6.

Fixes #11

### CODE REVIEW

File: packages/lokse/src/commands/update.ts [hunk 1]
```diff
@@ -53,6 +59,7 @@ class Update extends Base {
     const languages = flags.languages?.split(",") ?? this.conf?.languages;
     const column = flags.col ?? this.conf?.column;
     const format = flags.format ?? this.conf?.format ?? defaultFormat;
+    const sheets = flags.sheets?.split(",") ?? this.conf?.sheets ?? undefined;
```
Comment (jstorm31) [inline packages/lokse/src/commands/update.ts:62]: Is `?? undefined` at the end necessary if `this.conf?.sheets` gives `undefined` any way?

File: packages/lokse/src/core/reader/worksheet-reader.ts [hunk 2]
```diff
@@ -48,6 +58,18 @@ class WorksheetReader {
       this.shouldUseWorksheet(worksheet)
     );
 
+    if (worksheets.length === 0) {
+      let message = `Could find any sheets`;
```
Comment (jstorm31) [inline packages/lokse/src/core/reader/worksheet-reader.ts:62]: Typo: `Couldn't`

**Review by jstorm31 — APPROVED** [review]

File: packages/lokse/src/commands/update.ts [hunk 3]
```diff
@@ -43,6 +43,12 @@ class Update extends Base {
       options: outputFormats,
       description: `output format. Default is ${defaultFormat}.`,
     }),
+    sheets: flags.string({
+      char: "s",
+      name: "sheets",
+      description:
+        "sheets to get translations from. Name or list of names, comma sepaarated. For example ",
```
Comment (baierjak) [inline packages/lokse/src/commands/update.ts:50]: sepaarated

Comment (baierjak) [reply packages/lokse/src/commands/update.ts:50]: "Described sheets to get translations from. Name or a list of names, comma separated. For example ['Translations 1', 'Translations 2']"

File: packages/lokse/src/commands/update.ts (hunk 1 above)
Comment (baierjak) [reply packages/lokse/src/commands/update.ts:62]: Maybe in **this.conf?.sheets** could be some unwanted value?

File: packages/lokse/src/core/reader/__tests__/worksheet-reader.test.ts [hunk 4]
```diff
@@ -7,7 +7,7 @@ const createWorksheet = (title: string, index = 1) => ({
 });
 
 describe("WorksheetReader.shouldUseWorksheet", () => {
-  it("should use worksheet when empty or null or start", () => {
+  it("should use worksheet when empty or null or asterisk", () => {
```
Comment (baierjak) [inline packages/lokse/src/core/reader/__tests__/worksheet-reader.test.ts:10]: or obelisk :trollface:

**Review by jancama2 — APPROVED** [review]

File: packages/lokse/src/commands/update.ts (hunk 1 above)
Comment (horaklukas) [reply packages/lokse/src/commands/update.ts:62]: Thats true, we don't need  `undefined` here 👍

Check for valid filter value is on line 80 😉

File: packages/lokse/src/core/reader/__tests__/worksheet-reader.test.ts (hunk 4 above)
Comment (horaklukas) [reply packages/lokse/src/core/reader/__tests__/worksheet-reader.test.ts:10]: 😁
https://en.wikipedia.org/wiki/Asterisk here you are you little troll 😄

File: packages/lokse/src/commands/update.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (horaklukas) [reply packages/lokse/src/commands/update.ts:50]: yep 🙈

**Pull request merged**